appletree/plugin.py

```diff
@@ -27,8 +27,10 @@ class Plugin():
     # Set using the takes_config decorator
     takes_config = immutabledict()
 
-    def __init__(self):
+    def __init__(self, llh_name):
```

Comment:
Not sure if this is a good idea to require llh_name in the class lower than likelihood. Maybe give some default value?

Reply:
That is a good idea.